src/crypto/openssl/cose_sign.cpp

```diff
@@ -94,6 +94,28 @@ namespace ccf::crypto
     }
   }
 
+  COSEParametersFactory cose_params_cwt_map(const CWTMap& m)
```

Comment:
```suggestion
  COSEParametersFactory cose_params_cwt_map_str_int(const CWTMap& m)
```

perhaps, for clarity